aconcli/service/aconclient_http.go

```diff
@@ -0,0 +1,465 @@
+package service
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"mime/multipart"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+)
+
+const (
+	endpointBlob     = "/api/v1/blob"
+	endpointManifest = "/api/v1/manifest"
+	endpointFinalize = "/api/v1/manifest/finalize"
+	endpointStart    = "/api/v1/container/start"
+	endpointExec     = "/api/v1/container/exec"
+	endpointInspect  = "/api/v1/container/inspect"
+	endpointReport   = "/api/v1/container/report"
+	endpointKill     = "/api/v1/container/kill"
+	endpointRestart  = "/api/v1/container/restart"
+
```

Comment:
Just a thought, shall we use `/api/v1/container/<cid>/<command>` instead of `/api/v1/container/<command>&cid=<cid>`? `/api/v1/container/inspect` can continue to return all containers, while `/api/v1/container/<cid>/inspect` returns only the specified container.

Reply:
fixed